facebook_scraper/page_iterators.py

```diff
@@ -283,6 +283,8 @@ def get_next_page(self) -> Optional[URL]:
             if match:
                 value = match.groups()[0]
                 return value.encode('utf-8').decode('unicode_escape').replace('\\/', '/')
+    def get_page(self) -> Page:
```

Comment:
I got this error:
File <tokenize>:9
    def get_page(self) -> Page:
    ^
IndentationError: unindent does not match any outer indentation level